### PR TITLE
GeometryInfo<dim>::unit_normal_vector 

### PR DESCRIPTION
**Pull request by rezarastak**
Added a useful constant to `GeometryInfo` and used it to simplify a function in `mapping_cartesian.cc`. It also showcases the usefulness of `consexpr Tensor`.

### CODE REVIEW

File: source/fe/mapping_cartesian.cc [hunk 1]
```diff
@@ -289,50 +289,9 @@ MappingCartesian<dim, spacedim>::compute_fill(
...
-                        normals[face_no]);
-              break;
-            }
-
-          default:
-            Assert(false, ExcNotImplemented());
-        }
+      std::fill(normal_vectors.begin(),
+                normal_vectors.end(),
+                Point<dim>{GeometryInfo<dim>::unit_normal_vector[face_no]});
```
Comment (bangerth) [inline source/fe/mapping_cartesian.cc]: I *think* that we changed the type of the `normal_vectors` array to `Tensor<1,dim>` at some point in the past, even though this was not reflected in the initializers in the old code you are deleting here. If true, you should be able to get away without the cast in the last line here.

Comment (rezarastak) [reply source/fe/mapping_cartesian.cc]: Yes, you are right. I fixed it.

Comment (bangerth) [reply source/fe/mapping_cartesian.cc]: Thanks for checking!

**Pull request merged**